src/qt/bitcoin.cpp

```diff
@@ -298,9 +303,9 @@ void BitcoinCore::shutdown()
     try
     {
         qDebug() << __func__ << ": Running Shutdown in thread";
-        Interrupt(threadGroup);
+        FIXME_IMPLEMENT_IPC(Interrupt(threadGroup));
         threadGroup.join_all();
```

Comment:
> The thread group is completely remote in the case of IPC, isn't it?

Yes the change I'm working on now adds `threadGroup` and `scheduler` members to `NodeServer` in `ipc/server.cpp`, removing the current instances in `BitcoinCore`.